nanocloud/routes/users/users.go

```diff
@@ -238,18 +258,24 @@ func UpdatePassword(req router.Request) (*router.Response, error) {
 
 	if !exists {
 		return router.JSONResponse(404, hash{
-			"error": "User not found",
+			"error": [1]hash{
+				hash{
+					"detail": "User not found",
+				},
+			},
 		}), nil
 	}
 
-	err = users.UpdateUserPassword(userId, user.Password)
+	err = users.UpdateUserPassword(userId, user.Data.Password)
 	if err != nil {
 		log.Errorf("Unable to update user password: %s", err.Error())
 		return nil, err
 	}
 
 	return router.JSONResponse(200, hash{
-		"success": true,
+		"data": hash{
```

Comment:
We should return the user informations here: http://jsonapi.org/format/#crud-updating-responses-200

"If a server accepts an update but also changes the resource(s) in ways other than those specified by the request (for example, updating the updated-at attribute or a computed sha), it MUST return a 200 OK response. The response document MUST include a representation of the updated resource(s) as if a GET request was made to the request URL."